nbgrader/validator.py

```diff
@@ -258,12 +268,12 @@ def _get_passed_cells(self, nb):
 
             # if it's a grade cell, the check the grade
             if utils.is_grade(cell):
-                score, max_score = utils.determine_grade(cell)
+                score, max_score = utils.determine_grade(cell, self.log)
 
                 # it's a markdown cell, so we can't do anything
                 if score is None:
                     pass
-                elif score >= max_score:
+                elif score == max_score:
```

Comment:
Probably better to keep this as it was, in case the score is greater than the max score that also shouldn't count as failed:

```suggestion
                elif score >= max_score:
```

Reply:
I can change that, but then also need to update `get_partial_grade`, which currently returns a ValueError when score > max_score. Should I revert that back to a warning again?